internal/db/fetcher/versioned.go

```diff
@@ -408,7 +408,7 @@ func (vf *VersionedFetcher) merge(c cid.Cid) error {
 	}
 
 	// handle subgraphs
-	for _, l := range block.AllLinks() {
+	for _, l := range block.Links {
```

Comment:
nitpick: should we update the code comment above this line to explain why `Links` is used?